index.js

```diff
@@ -9,38 +13,42 @@ class DuplicateCompatError extends Error {
   }
 }
 
-function load() {
-  // Recursively load one or more directories passed as arguments.
+/**
+ * Recursively load one or more directories passed as arguments.
+ *
+ * @param {string[]} dirs The directories to load
+ * @returns {object} All of the browser compatibility data
+ */
+function load(...dirs) {
   let dir,
     result = {};
 
-  function processFilename(fn) {
-    const fp = path.join(dir, fn);
-    let extra;
+  for (dir of dirs) {
+    dir = path.isAbsolute(dir) ? dir : path.resolve(__dirname, dir);
+    const files = fs.readdirSync(dir);
+    for (const fn of files) {
+      const fp = path.join(dir, fn);
```

Comment:
```suggestion
      const abspath = path.resolve(__dirname, dir, entry);
```

Yes, I'm suggesting we rename `files` to `entries`, because they might not be files.